File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.cc
#include <algorithm>
#include <iostream>
#include <numeric>
#include <vector>
using namespace std;

int main(void) {
    int angle;
    vector<int> num;
    for (int i = 0; i < 3; i++) {
        cin >> angle;
        num.push_back(angle);
    }
    sort(num.begin(), num.end(), less<int>());
    if (accumulate(num.begin(), num.end(), 0) == 180) {
        if (num[0] == 60)
            cout << "Equilateral";
        else
            cout << (num[0] == num[1] or num[1] == num[2] ? "Isosceles" : "Scalene");
    } else {
        cout << "Error";
    }
    return 0;
}

File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.py
num=[]
for i in range(3):
    num+=[int(input())]
num.sort()
if sum(num)==180:
    if num[0]==60:
        print("Equilateral")
    else:
        print("Isosceles" if num[0]==num[1] or num[1]==num[2] else "Scalene")
else:
    print("Error")